packages/desktop-client/src/components/budget/ExpenseCategory.tsx

```diff
@@ -35,6 +39,7 @@ type ExpenseCategoryProps = {
 
 export function ExpenseCategory({
   cat,
+  catGroup,
```

Comment:
I would suggest accepting an isHidden prop instead. Same with the sidebar category.

Reply:
I actually had it that way originally and decided to go this route instead.  That other way, the logic winds up having to be the same 'above' anywhere that uses that component.  It seemed to make more sense to keep it 'low' to avoid possible duplication.

Reply:
See: https://github.com/actualbudget/actual/pull/2582/files#diff-1a3df24d67ec18144fd290e1950428038465bfc531ad64e953ff27753d55f1f1R107